packages/common-all/src/analytics.ts

```diff
@@ -3,6 +3,7 @@ export enum VSCodeEvents {
   ServerCrashed = "ServerCrashed",
   InitializeWorkspace = "InitializeWorkspace",
   Install = "Install",
+  NewInstanceInstall = "NewInstanceInstall",
```

Comment:
instead of creating a new event, can we use "Install" event but with the property, "newInstance" set to true?

Reply:
fixed, and also recorded in airtable